packages/frontend2/src/app/(legacy)/scaling-next/projects/[slug]/tvl-breakdown/_components/tables/table-sum.tsx

```diff
@@ -0,0 +1,18 @@
+import { formatNumberWithCommas } from '~/utils/format-number'
+
+interface TableSumProps {
+  amount: number
+}
+
+export function TableSum(props: TableSumProps) {
+  return (
+    <div className="mt-3 flex w-full flex-row-reverse self-end border-t border-t-black/10 py-2 pr-0 text-base font-medium dark:border-t-white/25 md:pr-4">
+      <div>
+        <span className="text-gray-500 dark:text-gray-50">Total:&nbsp;</span>
+        <span className="text-pink-800 dark:text-pink-200">
+          ${formatNumberWithCommas(props.amount)}
+        </span>
+      </div>
+    </div>
+  )
+}
```

Comment:
Instead of using this one we should use `footer` property of a column.
https://tanstack.com/table/latest/docs/api/core/column-def#footer

Reply:
use case is valid but footer will follow overflow on mobile, so to see the actual sum user has to scroll to the very right